GitCommands/Git/GitModule.cs

```diff
@@ -2185,13 +2185,8 @@ public Patch GetSingleDiff(string @from, string to, string fileName, string oldF
 
             //fix refs slashes
             from = from.ToPosixPath();
-            to = to.ToPosixPath();
-            string commitRange = string.Empty;
-            if (!to.IsNullOrEmpty())
-                commitRange = "\"" + to + "\"";
-            if (!from.IsNullOrEmpty())
-                commitRange = string.Join(" ", commitRange, "\"" + from + "\"");
-
+            to = to == null ? "":to.ToPosixPath();
+            string commitRange = RevisionDiffProvider.Get(from, to);
```

Comment:
I don't to see any static coupling, especially in GitModule. This monolith needs to be broken down and tested, static coupling get in the way.
Remember, I asked to make instance class in https://github.com/gitextensions/gitextensions/pull/4135#discussion_r150400146